src/styles/blocks/_header.scss

```diff
@@ -0,0 +1,54 @@
+.header {
```

Comment:
Front of the bike could be positioned exactly, you see on mobile the front tire is centered

Reply:
It is, actually. Check it at 320px x 480px as it is shown in the mockup